packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -318,5 +320,7 @@ export class ARRenderer extends EventDispatcher {
       // this.threeRenderer.clearDepth();
       this.threeRenderer.render(this.scene, this.camera);
     }
+
+    this.scene.environment = null;
```

Comment:
I think clearing `scene.environment` is safer to prevent memory leak.

Reply:
I think it might be better to put this in the `[$postSessionCleanup]` method. WDYT?

https://github.com/google/model-viewer/blob/e835eb3bef53a30ae9af1f3f4f43a179f80b5e37/packages/model-viewer/src/three-components/ARRenderer.ts#L186

Reply:
I think that sounds reasonable. I'll update tomorrow.

Reply:
Updated, thanks